one-ui/src/main/java/com/marklogic/hub/oneui/controllers/EnvironmentController.java

```diff
@@ -122,4 +126,16 @@ public JsonNode reset() {
         obj.put("reset", true);
         return obj;
     }
+
+    @RequestMapping(value = "/project-info", method = RequestMethod.GET)
+    @ResponseBody
+    public JsonNode getProjectInfo(HttpSession session) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        ObjectNode resp = objectMapper.createObjectNode();
+        resp.put("dhfVersion", versions.getHubVersion());
+        resp.put("projName", hubConfig.getHubProject().getProjectName());
```

Comment:
Let's spell these out, there's no benefit in shortening them - so "projectName" and "projectPath" (I think "path" is better than "dir" here, particularly because the output of getProjectDir is a Path (which means that method should really be called getProjectPath, but oh well)).